package.json

```diff
@@ -76,8 +82,8 @@
     "prepare": "npm run lint && npm run test && npm run build",
     "build": "babel --out-dir build src --ignore src/__tests__",
     "build:watch": "npm run build -- --watch",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "jest --testPathIgnorePatterns '/helpers/'",
+    "test:watch": "jest --testPathIgnorePatterns '/helpers/' --watch",
```

Comment:
Instead of having to specify to ignore `/helpers/`, could the setup file be in a folder adjacent to `/src` (maybe call it `/setup` or something)? See my comment below for more info